src/classes/TDTM_ProcessControl.cls

```diff
@@ -120,4 +132,44 @@ public class TDTM_ProcessControl {
             }
         }
     }
+
+    public Boolean allowRecursionInTests() {
```

Comment:
_Non-blocking_

I initially thought this was a command method. It still sounds like a command to me. What're your thoughts on making query methods sound more question-like? `isRecursionAllowedInTests` vs `allowRecursionInTests`